Dockerfile

```
@@ -0,0 +1,28 @@
+FROM node:lts-bullseye as build-deps
+
+EXPOSE 3000
+
+ENV DIRPATH /opt/app
+ENV REACT_APP_REAL_SERVER_BASE=https://smart-health-links-server.cirg.washington.edu/api
+ENV NODE_ENV production
+
+WORKDIR ${DIRPATH}
+
+COPY ui ${DIRPATH}/ui
+COPY client ${DIRPATH}/client
+
+WORKDIR ${DIRPATH}/ui
+
+RUN npm ci
```

Nice, wasn't [aware of `ci`](https://docs.npmjs.com/cli/v8/commands/npm-ci/) (vs `install`)! 

However, one of `ci`'s other aliases would be more descriptive

```suggestion
RUN npm clean-install
```